docker/1.0-1/final/Dockerfile.cpu

```diff
@@ -8,9 +8,9 @@ RUN python -m pip install -r /requirements.txt && \
     rm /requirements.txt
 
 COPY dist/sagemaker_sklearn_container-2.0-py3-none-any.whl /sagemaker_sklearn_container-2.0-py3-none-any.whl
-# https://github.com/googleapis/google-cloud-python/issues/6647
-RUN rm -rf /miniconda3/lib/python3.7/site-packages/numpy-1.19.4.dist-info && \
-    pip install --no-cache /sagemaker_sklearn_container-2.0-py3-none-any.whl && \
+RUN rm /miniconda3/lib/python3.8/site-packages/**/REQUESTED && \
```

Comment:
Should we use `-f` here in case the file doesn't exist going forward?

Reply:
I think it is fine if we leave it as is. These files are part of the PyPA specs, and reasons for why these files may no longer exist would relate to us not using conda or not having a requirements file. Even if in that unlikely scenario, I think it is fine for this to fail at build time on our side to force us to remove this.